Fix ld_libdir_mangohud used by the mangohud shell script to locate the
libMangoHud.so lib.
--- a/src/meson.build
+++ b/src/meson.build
@@ -3,10 +3,10 @@
 # Needs prefix for configure_file()
 if get_option('append_libdir_mangohud')
   libdir_mangohud = join_paths(get_option('prefix'), get_option('libdir'), 'mangohud')
-  ld_libdir_mangohud = get_option('prefix') + '/\$LIB/mangohud/'
+  ld_libdir_mangohud = join_paths(get_option('prefix'), get_option('libdir'), 'mangohud') + '/'
 else
   libdir_mangohud = join_paths(get_option('prefix'), get_option('libdir'))
-  ld_libdir_mangohud = get_option('prefix') + '/\$LIB/'
+  ld_libdir_mangohud = join_paths(get_option('prefix'), get_option('libdir')) + '/'
 endif
 
 conf_data = configuration_data()
